client/service/src/task_manager/mod.rs

```diff
@@ -51,10 +73,10 @@ pub struct SpawnTaskHandle {
 }
 
 impl SpawnTaskHandle {
-	/// Spawns the given task with the given name and an optional group name.
-	/// If group is not specified `DEFAULT_GROUP_NAME` will be used.
+	/// Spawns the given task with the given name and a `GroupName` type which can be an
+	/// actual(specified) group name. If group is not specified `DEFAULT_GROUP_NAME` will be used.
```

Comment:
I think the docs on `GroupName` should be enough.
```suggestion
	/// Spawns the given task with the given name and a group name.
	/// If group is not specified `DEFAULT_GROUP_NAME` will be used.
```